libs/utils/energy_model.py

```diff
@@ -1012,6 +1033,11 @@ def from_target(cls, target):
                   reported by the target.
         """
         _log = logging.getLogger('EMReader')
+        yaml.register_class(ActiveState)
```

Comment:
According to the doc, explicit registration is not necessary if the classes are decorated with @yaml_object(yaml)

http://yaml.readthedocs.io/en/latest/dumpcls.html